grails-app/conf/RoleFilters.groovy

```diff
@@ -57,13 +64,15 @@ class RoleFilters {
                 }
 
                 // Authorized users
+                def missAuthenticated = !userService.isUserInRole(session.user, RoleType.ROLE_AUTHENTICATED)
```

Comment:
How about changing naming of all these variables into:
`isNotAuthenticated`
`isNotBrowser`
`isNotManager`
`isNotAdmin`
`isNotSuperuser`
`hasNotRoleInvoice`
`isNotRequestor`

Reply:
ok